Create nominees

Msgl opened the pull request.

nathanbaleeta reviewed (approved):
Looks good!

nathanbaleeta reviewed (changes requested):
Thank you @Msgl the submission looks good! Instead of submitting all your nominated projects(`Open Source Ecology`, `OnSSET` & `VIAME` etc) on separate pull requests, could you kindly combine them into a batch and update this pull request for easier tracking of your work when we are doing a review. Feel free to close PRs #428, #429 & #430 after merging into #431. Thanks.

Msgl commented:
Thank you for the feedback @nathanbaleeta! I merged the files to this PR and I closed the extra PR's as you suggested. I also updated the title of this PR and added an extra file. Please let me know if everything is ok now.

nathanbaleeta commented on focus.json:
```diff
@@ -0,0 +1,37 @@
+  "type": [
+    "software"
+  ],
+  "repositoryURL": "https://github.com/UnionVMS",
```
@Msgl could you be more explicit and provide a link to a specific repo within [UnionVMS](https://github.com/UnionVMS) organization. Thanks.

nathanbaleeta commented on nominees/opensourceecology.json:
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "OpenSourceEcology",
```
@Msgl could you update the name field to "Open Source Ecology" as represented on the website. Thanks.

nathanbaleeta commented on nominees/opensourceecology.json:
```diff
@@ -0,0 +1,35 @@
+  "license": [
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "https://github.com/OpenSourceEcology/cross-platform-python-gui"
```
@Msgl kindly update the `licenseURL` to point to a specific page within the same repo referencing the [license](https://github.com/OpenSourceEcology/cross-platform-python-gui/blob/master/LICENSE) . Thanks.

nathanbaleeta commented on nominees/opensourceecology.json:
```diff
@@ -0,0 +1,35 @@
+  "repositoryURL": "https://github.com/OpenSourceEcology/cross-platform-python-gui",
+  "organizations": [
+    {
+      "name": "OpenSourceEcology",
```
@Msgl could you update the name field to "Open Source Ecology" as represented on the website. Thanks.

nathanbaleeta commented on nominees/viame.json:
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "VIAME",
+  "description": "A DIY AI system for analyzing imagery and video, orginally targeting marine species analysis.",
```
@Msgl could you update the DIY abbreviation to the full form, do-it-yourself in the `description` field. Thanks.

nathanbaleeta commented on nominees/viame.json:
```diff
@@ -0,0 +1,36 @@
+  "license": [
+    {
+      "spdx": "BSD-3-Clause",
+      "licenseURL": "https://github.com/VIAME/VIAME"
```
@Msgl could you update the `licenseURL` to point to a more explicit [URL](https://github.com/VIAME/VIAME/blob/master/LICENSE.txt) within the same repo. Thanks.

nathanbaleeta commented on focus.json:
```diff
@@ -0,0 +1,37 @@
+  "type": [
+    "software"
+  ],
+  "repositoryURL": "https://github.com/UnionVMS/UVMS-MobileTerminalModule-APP",
```
@Msgl could you please justify how you settled for this [repo](https://github.com/UnionVMS/UVMS-MobileTerminalModule-APP) of all the `repos` under [UnionVMS](https://github.com/UnionVMS) Github org. Thanks.

Msgl replied:
As I understand by going through their repository, many repos are linked to one another, and I could not find a clear structure where it shows that one repo is the "main" one. That was the reason why I initially added their GitHub URL without selecting a specific repo. 

I went through their manual installation documentation at https://focusfish.atlassian.net/wiki/spaces/UVMS/pages/15925281/Liquibase+setup, where a repo UVMS-MobileTerminalModule-DB is mentioned for installing and using the application on a mobile terminal. I searched through their repo and found the UVMS-MobileTerminalModule-APP which seems to have the same README.md file. So if I have understood correctly, and this is indeed the repo that a person can clone and use for mobile monitoring of fish resources, I believe this is the appropriate repo as it also provides usability for the user.

nathanbaleeta replied:
@Msgl sounds convincing. Let's proceed with this then.

nathanbaleeta commented on focus.json:
```diff
@@ -0,0 +1,37 @@
+  "license": [
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "http://www.focus.fish/"
```
@Msgl could you update the license to point to this explicit [link](https://github.com/UnionVMS/UVMS-MobileTerminalModule-APP/blob/dev/LICENSE). Thanks.

Msgl commented:
I have updated the changes requested, thank you for the feedback @nathanbaleeta ! Please let me know if everything is ok.

nathanbaleeta reviewed (approved):
Thanks @Msgl, looks good to me.
